smart-hadoop-support/smart-hadoop/src/main/java/org/smartdata/hdfs/HadoopUtil.java

```diff
@@ -220,4 +221,32 @@ public DFSClient run() throws Exception {
     }
   }
 
+  public static String translateStoragePoliceId2Name(int id) {
+    if (id == 15) {
+      return "Lazy_Persist";
+    }
+
+    if (id == 12) {
+      return "All_SSD";
+    }
+
+    if (id == 10) {
+      return "One_SSD";
+    }
+
+    if (id == 7) {
+      return "Hot";
+    }
+
+    if (id == 5) {
+      return "Warm";
+    }
+
+    if (id == 2) {
+      return "Cold";
+    }
+
+    return "";
```

Comment:
A reversed method, i.e., string->id, is also necessary. Meanwhile, make sure all string is correct.